packages/in_app_purchase/in_app_purchase_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.0+19
```

Comment:
Nit: As a new feature it should be incrementing the "minor" version (which for <1.0 in Dart convention is the third instead of the second), so this should be `0.3.1`.

Reply:
Fixed, but I do think that it is weird that0.3.0+19 is less than 0.3.1